logstash/class-logger.php

```diff
@@ -290,19 +291,17 @@ public function debug( string $feature, string $message, array $context = [] ) :
 	protected static function parse_params( array $params ) : array {
 		// Prepare data.
 		$default_params = [
-			'site_id'   => get_current_network_id(),                  // Required.
-			'blog_id'   => get_current_blog_id(),                     // Required.
-			'http_host' => strtolower( $_SERVER['HTTP_HOST'] ?? '' ), // phpcs:ignore -- Optional.
-
-			'severity'  => '',                                        // Optional.
-			'feature'   => '',                                        // Required.
-			'message'   => '',                                        // Required.
-
-			'user_id'   => get_current_user_id(),                     // Optional.
-
-			'extra'     => [],                                        // Optional.
-			'timestamp' => gmdate( 'Y-m-d H:i:s' ),                   // Required.
-			'index'     => 'log2logstash',                            // Required
+			'site_id'         => get_current_network_id(),                    // Required.
+			'blog_id'         => get_current_blog_id(),                       // Required.
+			'http_host'       => strtolower( $_SERVER['HTTP_HOST'] ?? '' ),   // phpcs:ignore -- Optional.
+			'severity'        => '',                                          // Optional.
+			'feature'         => '',                                          // Required.
+			'message'         => '',                                          // Required.
+			'user_id'         => get_current_user_id(),                       // Optional.
+			'extra'           => [],                                          // Optional.
+			'timestamp'       => gmdate( 'Y-m-d H:i:s' ),                     // Required.
+			'index'           => 'log2logstash',                              // Required,
+			'http_user_agent' => self::get_user_agent(),
```

Comment:
Just to keep the pattern of the rest (assuming it is optional):

```suggestion
			'timestamp'       => gmdate( 'Y-m-d H:i:s' ),                     // Required.
			'index'           => 'log2logstash',                              // Required.
			'http_user_agent' => self::get_user_agent(),                      // Optional.
```